engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -258,6 +258,40 @@ typedef enum {
   kFlutterSemanticsFlagIsRequired = 1 << 30,
 } FlutterSemanticsFlag;
 
+typedef struct {
+  bool hasCheckedState;
```

Comment:
Could we snake case these? Something like:

```suggestion
  bool has_checked_state;
```

Reply:
Yes -- these must use C/C++ style. So is_checked, has_checked_state, etc.